scipy-notebook/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN conda install --quiet --yes \
     'h5py=2.10.*' \
     'hdf5=1.10.*' \
     'ipywidgets=7.5.*' \
+    'ipympl=0.3.*'\
```

Comment:
Replaced by `ipympl=0.5.*`

Reply:
Thanks! I've independently verified that this indeed solves it.